tests/Oriflame.PolicyBuilder.Xml.Tests/Definitions/Common/ForwardRequestsTests.cs

```diff
@@ -8,22 +8,12 @@ namespace Oriflame.PolicyBuilder.Xml.Tests.Definitions.Common
     public class ForwardRequestsTests
     {
         [Theory]
-        [InlineData(5)]
-        public void CreatesCorrectPolicy(int timeoutSeconds)
+        [InlineData("5")]
+        public void CreatesCorrectPolicy(string timeoutSeconds)
         {
-            var basePolicy = new ForwardRequest(TimeSpan.FromSeconds(timeoutSeconds));
+            var basePolicy = new ForwardRequest(new Dictionary<string, string>());
             var xml = basePolicy.GetXml().ToString();
-            xml.Should().Be($"<forward-request timeout=\"{timeoutSeconds}\" />");
-        }
-        
-        [Theory]
-        [InlineData(5, true)]
-        [InlineData(3, false)]
-        public void CreatesCorrectBufferResponsePolicy(int timeoutSeconds, bool bufferResponse)
```

Comment:
I would leave this test as a partial one, just update the body to use the AttributeBuilder.

Reply:
This is already covered by `BackendSectionPolicyBuilderTests` but you are right, here we can cover the attribute builder logic.